registry/handlers/helpers.go

```diff
@@ -2,6 +2,7 @@ package handlers
 
 import (
 	"encoding/json"
+	"github.com/docker/distribution/registry/api/errcode"
```

Comment:
It's better to place non-stdlib imports after a space line. I don't insist on it, just to follow the same style across the project :)